helm/hello-world/values.schema.json

```diff
@@ -100,185 +201,423 @@
                 },
                 "tls": {
                     "type": "array",
+                    "title": "TLS",
+                    "description": "You will have to overwrite the hosts values to make it match your cluster's ingress subdomain, matching the names used as ingress hosts.",
                     "items": {
                         "type": "object",
                         "properties": {
                             "hosts": {
                                 "type": "array",
+                                "description": "Hostnames to use the TLS certificate for.",
                                 "items": {
-                                    "type": "string"
+                                    "type": "string",
+                                    "format": "hostname"
                                 }
                             },
                             "secretName": {
-                                "type": "string"
+                                "type": "string",
+                                "description": "Name of the secret containing the TLS certificate."
                             }
                         }
                     }
                 }
             }
         },
         "nameOverride": {
-            "type": "string"
+            "type": "string",
+            "title": "Name override",
+            "description": "If set, this name will be used instead of Chart.Name for resource naming.",
+            "default": ""
         },
         "nodeSelector": {
-            "type": "object"
+            "type": "object",
+            "title": "Node selector",
+            "description": "If specified, this app will only be deployed to nodes with matching labels.",
+            "additionalProperties": false,
+            "patternProperties": {
+                "^[a-zA-Z0-9/-_\\.]+$": {
+                    "$ref": "https://schema.giantswarm.io/labelvalue/v0.0.1"
+                }
```

Comment:
This is an example where we use the `$ref` key to point to an external schema.

https://schema.giantswarm.io/labelvalue/v0.0.1 is a schema we created and published exactly for this purpose.